src/larktools/evaluation.py

```diff
@@ -109,3 +108,16 @@ def eval_bracketed_arith_expr(node, env):
     child = get_children(node)[0]
     assert get_name(child) == "arith_expr"
     return eval_arith_expr(child, env)
+
+
+def eval_variable(node, env):
```

Comment:
Maybe we could add type hints too - what do you think?

Reply:
I added it on our todo list #2